pkg/env/dev.go

```diff
@@ -147,9 +153,46 @@ func newDev(ctx context.Context, log *logrus.Entry, instancemetadata instancemet
 		return nil, err
 	}
 
+	// TODO get the geneva certs for dev and put them into the secrets
+	b, err = ioutil.ReadFile("secrets/gcskey.pem")
+	if err != nil {
+		return nil, err
+	}
+
+	d.gcsKey, err = x509.ParsePKCS1PrivateKey(b)
+	if err != nil {
+		return nil, err
+	}
+
+	b, err = ioutil.ReadFile("secrets/gcscert.pem")
+	if err != nil {
+		return nil, err
+	}
+
+	d.gcsCert, err = x509.ParseCertificate(b)
+	if err != nil {
+		return nil, err
+	}
+
 	return d, nil
 }
 
+func (d *dev) GenevaLoggingConfig() *genevalogging.GenevaLoggingConfig {
+	// TODO these are from the v3 plugin, find out their real values.
```

Comment:
I need the correct values for all of these.

Reply:
took me a bit on the names.  So I have these values for rp logging but not for cluster logging. This is for cluster logging correct?

Reply:
yes, cluster logging

Reply:
@JackQuincy  suggest new names?

Reply:
@asalkeld these values are in the wiki, ping me if you don't have the link